### DISCUSSION TITLE
[Question] PYENV_VIRTUALENV_DISABLE_PROMPT=1' to simulate the behavior` on pyenv activate

### DISCUSSION THREAD

**Discussion by sla-te** [pyenv/pyenv#2107] [Q&A] [answered]
`pyenv-virtualenv: prompt changing will be removed from future release. configure `export
PYENV_VIRTUALENV_DISABLE_PROMPT=1' to simulate the behavior.`

upon doing pyenv activate <my_env_name>, is there a new handling, that is suggested for future releases?

**Comment by native-api** [comment, marked as answer]
I don't understand the issue.
Currently, Pyenv-Virtualenv prepends `(env_name) ` to shell prompt. This warning tells you that this behavior is going away sometime in the future. Before that happens, you can set the mentioned envvar to do that right now and silence the warning.

It doesn't affect the behavior of any Pyenv invocations, you don't need to change anything in your code to compensate or anything.

**Reply by sla-te** [reply]
Ah, ok, I didnt get the message that way  'prompt changing' was not clear to me and I thought I have to adapt to a change, thank you for clarifying.
